arangod/Aql/OptimizerRules.cpp

```diff
@@ -7471,6 +7470,35 @@ void arangodb::aql::geoIndexRule(Optimizer* opt,
     LimitNode* limit = nullptr;
     bool canUseSortLimit = true;
 
+    bool mustRespectIdxHint = false;
+    if (plan->hasForcedIndexHints()) {
+      auto enumerateColNode =
+          ExecutionNode::castTo<EnumerateCollectionNode const*>(node);
+      auto indexes = enumerateColNode->collection()->indexes();
+      VPackBuilder b;
+      b.openObject();
+      enumerateColNode->hint().toVelocyPack(b);
```

Comment:
Why not use `enumerateColNode->hint().hint()` to get list of index names directly?